xml/storage_multipath.xml

```diff
@@ -364,7 +288,8 @@
     that the <filename>initrd</filename> and the installed system behave the
     same regarding the root file system and all other file systems required to
     boot the system. If Multipath is enabled in the system, it also needs to be
-    enabled in the <filename>initrd</filename> and vice versa. See <xref
+    enabled in the <filename>initrd</filename> and vice versa. See
```

Comment:
Seems like this should say `enabled in the <filename>initrd</filename> file`

Reply:
`initrd` is not a settings file, though; it is a RAMdisk containing lots of files, supplied to the kernel.

Reply:
right, sorry.